miniapp/src/main/java/com/rakuten/tech/mobile/miniapp/storage/MiniAppSecureStorage.kt

```diff
@@ -63,20 +89,28 @@ internal class MiniAppSecureStorage(
                 if (createOrOpenAndUnlockDatabase()) {
                     onSuccess()
                 }
-            } catch (e: SQLException) {
+            } catch (e: RuntimeException) {
                 onFailed(MiniAppSecureStorageError.secureStorageIOError)
             }
         }
     }
 
-    fun getDatabaseUsedSize(
+    internal fun getDatabaseUsedSize(
```

Comment:
I think, we don't need to specify `internal` as the whole class in already `internal`.

Reply:
Addressed.